lib/round.rb

```diff
@@ -0,0 +1,50 @@
+class Round
+  attr_reader :deck,
+              :turns
+
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+  end
+
+  def current_card
+    @deck.cards[@turns.count]
+  end
+
+  def take_turn(answer)
+    new_turn = Turn.new(answer, @deck.cards[@turns.count])
+    @turns << new_turn
+    return new_turn
+  end
```

Comment:
On line 11, when you have `@deck.cards[@turns.count]`, this is code that you have in your `current_card` method. So, you could replace that part of line 11 with a method call to `current_card` to dry up your code.